use_cases/eluc/app/app.py

```diff
@@ -18,44 +18,35 @@
 import dash_bootstrap_components as dbc
 
 from data import constants
-from data.eluc_data import ELUCEncoder
 import app.constants as app_constants
 from app import utils
-from prescriptors.nsga2.torch_prescriptor import TorchPrescriptor
 
 app = Dash(__name__,
            external_stylesheets=[dbc.themes.BOOTSTRAP, dbc.icons.BOOTSTRAP],
            prevent_initial_callbacks="initial_duplicate")
 server = app.server
 
 df = pd.read_csv(app_constants.DATA_FILE_PATH, index_col=app_constants.INDEX_COLS)
-df.rename(columns={col + ".1": col for col in app_constants.INDEX_COLS}, inplace=True)
 COUNTRIES_DF = regionmask.defined_regions.natural_earth_v5_0_0.countries_110.to_dataframe()
 
-# Prescriptor list should be in order of least to most change
+# Load pareto df
 pareto_df = pd.read_csv(app_constants.PARETO_CSV_PATH)
-pareto_df = pareto_df.sort_values(by="change", ascending=True)
+pareto_df.sort_values(by="change", inplace=True)
 prescriptor_list = list(pareto_df["id"])
 
-encoder = ELUCEncoder.from_json(app_constants.PRESCRIPTOR_PATH / "fields.json")
-# TODO: Stop hard-coding candidate params -> make cand config file?
-candidate_params = {
-    "in_size": len(constants.CAO_MAPPING["context"]),
-    "hidden_size": 16,
-    "out_size": len(constants.RECO_COLS)
-}
-prescriptor = TorchPrescriptor(None, encoder, None, 1, candidate_params)
+# Load prescriptors
+prescriptor_manager = utils.load_prescriptors()
```

Comment:
Just use a single PrescriptorManager object now that can hold all the individual LandUsePrescriptors. This allows us to in the future add Heuristics too since they implement Prescriptor